lib/ansible/modules/packaging/os/yum.py

```diff
@@ -1117,23 +1147,11 @@ def ensure(module, state, pkgs, conf_file, enablerepo, disablerepo,
         # should be caught by AnsibleModule argument_spec
         module.fail_json(msg="we should never get here unless this all"
                 " failed", changed=False, results='', errors='unexpected state')
-
     return res
 
 
 def main():
 
-    # state=installed name=pkgspec
-    # state=removed name=pkgspec
-    # state=latest name=pkgspec
-    #
-    # informational commands:
-    #   list=installed
-    #   list=updates
-    #   list=available
-    #   list=repos
-    #   list=pkgspec
-
```

Comment:
Leave this, at least for now.  there's no other listing of the two types of commands (and the arguments that list takes aren't even i nthe documentatoin :-(